tests/event/event_simulation_tester.py

```diff
@@ -41,7 +41,7 @@ def _create_artifacts(self) -> None:
         artifacts = self.simulator.create_artifacts(builder)
 
         assert peer_id.id is not None
-        self.peer_id: str = peer_id.id
+        self.peer_id: str = str(peer_id.id)
```

Comment:
`self.peer_id` is used heavily throughout the test for asserting against str versions of peer-id's, because it's what ends up in the event entries, using `str` is really convenient, if it was `self.peer_id: PeerId` I'd end up having to refactor all uses to to do `str(self.peer_id)` instead, I think there's little benefit to that.

Reply:
Got it. Thanks!